firmware/library/L1_Drivers/pwm.hpp

```diff
@@ -1,9 +1,6 @@
 #pragma once
 
 #include <cstdint>
-#include <cstdio>
-#include "L0_LowLevel/LPC40xx.h"
-#include "L1_Drivers/pin_configure.hpp"
 
 class PWMInterface
 {
```

Comment:
I would think that you would also need an `Initialize()` function as well. Bare in mind that constructors should not be altering hardware.

Reply:
I have this function too on line 8.